src/participant/participant.ts

```diff
@@ -1201,6 +1202,66 @@ export default class ParticipantController {
     });
   }
 
+  async exportCodeToPlayground(): Promise<boolean> {
+    const activeTextEditor = vscode.window.activeTextEditor;
+    if (!activeTextEditor) {
+      void vscode.window.showErrorMessage('Active editor not found.');
+      return Promise.resolve(false);
+    }
+
+    const sortedSelections = Array.from(activeTextEditor.selections).sort(
+      (a, b) => a.start.compareTo(b.start)
+    );
+    const selectedText = sortedSelections
+      .map((selection) => activeTextEditor.document.getText(selection))
+      .join('\n');
+
+    const code =
+      selectedText || activeTextEditor.document.getText().trim() || '';
+
+    try {
+      const progressResult = await vscode.window.withProgress(
+        {
+          location: vscode.ProgressLocation.Notification,
+          title: 'Exporting code to a playground...',
+          cancellable: true,
+        },
+        async (progress, token) => {
+          const messages = EportToPlaygroundPrompt.buildMessages(code);
+          return await this.getChatResponseContent({
+            messages,
+            token,
+          });
+        }
+      );
+
+      if (progressResult?.includes("Sorry, I can't assist with that.")) {
+        void vscode.window.showErrorMessage("Sorry, I can't assist with that.");
+        return Promise.resolve(false);
+      }
+
+      if (progressResult) {
+        const runnableContent = getRunnableContentFromString(progressResult);
+        if (progressResult) {
```

Comment:
Should this be `runnableContent`? Related is the question on code block in the prompt.
```suggestion
        if (runnableContent) {
```